src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,26 +5,19 @@ sidebar_order: 3
 
 {% include learn-sdk.md platform="javascript" %}
 
-All our JavaScript-related SDKs provide the same API still there are some differences between them which this section of the docs explains.
-
-## Browser Compatibility
-
-Our JavaScript SDK supports all major browsers. In older browsers, error reports may have a degraded level of detail – for example, missing stack trace data or missing source code column numbers.
-
 The table below shows supported browsers:
 
 ![Sauce Test Status](https://saucelabs.com/browser-matrix/sentryio.svg)
 
 {% capture __alert %}
 Our SDK needs a polyfill for `Promise` in older browsers like IE 11 and below. 
 Please add this script tag below before loading our SDK. 
-If you are using the npm package please use a polyfill for `Promise` respectively.
+If you are using the npm package, please use a polyfill for `Promise` respectively.
```

Comment:
Is there a Promise polyfill package we recommend?

Reply:
Isn't it just https://cdn.polyfill.io/v2/polyfill.min.js?features=Promise below?

Reply:
Yes, exactly this one from `polyfill.io`

Reply:
I'll reword the Note to make it more clear.